whipper/program/cdparanoia.py

```diff
@@ -196,7 +196,7 @@ def getTrackQuality(self):
 
         # don't go over a 100%; we know cdparanoia reads each frame at least
         # twice
-        return min(frames * 2.0 / reads, 1.0)
+        return min(frames * 2.0 / reads, 1.0) if reads else 0
```

Comment:
BTW, I haven't checked: is it OK to have reads with a value of 0? (is there any underlying bug?)